Fix an erroneous structure declaration for FreeBSD.

===================================================================
RCS file: /home/jdp/m3-cvs/m3/m3core/src/C/FreeBSD2/Cstdio.i3,v
retrieving revision 1.1.1.1
diff -u -r1.1.1.1 Cstdio.i3
--- m3/m3core/src/C/FreeBSD2/Cstdio.i3.orig	1996/09/24 05:21:57	1.1.1.1
+++ m3/m3core/src/C/FreeBSD2/Cstdio.i3	1996/09/24 05:32:37
@@ -55,6 +55,7 @@
         (* Unix stdio files get aligned to block boundaries on fseek() *)
         blksize : int;        (* stat.st_blksize (may be != _bf._size) *)
         offset  : off_t;      (* current lseek offset *)
+        pad1    : int;        (* assume high 4 bytes of offset are 0 *)
  
         END;
 
